lib/api/decorators/single.rb

```diff
@@ -51,6 +51,29 @@ def initialize(model, context = {})
                exec_context: :decorator,
                render_nil: false
 
+      def self.self_link(path, title_getter: -> (*) { represented.name })
+        link :self do
+          link_object = { href: api_v3_paths.send(path, represented.id) }
+          link_object[:title] = instance_eval(&title_getter)
+
+          link_object
+        end
+      end
+
+      def self.linked_property(property,
+                               path: property,
+                               backing_field: property,
```

Comment:
Am I missing something or **do** I already set property as default for both?

So you propose `association` as name, right?